lib/rubocop/cop/rails/env_local.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Checks for usage of `Rails.env.development? || Rails.env.test?` which
+      # can be replaced with `Rails.env.local?`, introduced in Rails 7.1.
+      #
+      # @example EnforcedStyle: local (default)
+      #
+      #   # bad
+      #   Rails.env.development? || Rails.env.test?
+      #
+      #   # good
+      #   Rails.env.local?
+      #
+      # @example EnforcedStyle: no_local
+      #
+      #   # bad
+      #   Rails.env.local?
+      #
+      #   # good
+      #   Rails.env.development? || Rails.env.test?
```

Comment:
The `EnforcedStyle: no_local` option looks unnecessary. So, it would be sufficient to provide only the default.

Reply:
My justification was that authors may prefer to be explicit (or perhaps they need to support an old Rails version, in the case of a gem author) and not want to use `local?`. Normally this would be covered by updating the `UnknownEnv` config, except that #894's implementation always injects it in if the target Rails version is >= 7.1.

However, if you think that's too obscure, I can certainly remove the `EnforcedStyle` option.

Reply:
Done 👍